<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gproductos</class>
 <widget class="QDialog" name="Gproductos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestionar Productos</string>
  </property>
  <widget class="QDockWidget" name="informacion5">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>300</y>
     <width>112</width>
     <height>136</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>44</height>
    </size>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable</set>
   </property>
   <property name="windowTitle">
    <string>Informacion</string>
   </property>
   <widget class="QWidget" name="informacion5_2">
    <widget class="QLabel" name="informacionlabel5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>901</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Gestionar Productos</string>
   </property>
   <widget class="QTableView" name="lista5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>901</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>280</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="addbutton5">
       <property name="text">
        <string>Adicionar</string>
       </property>
       <property name="icon">
        <iconset resource="pandasoft.qrc">
         <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="modbutton5">
       <property name="text">
        <string>Modificar</string>
       </property>
       <property name="icon">
        <iconset resource="pandasoft.qrc">
         <normaloff>:/images/document-edit.png</normaloff>:/images/document-edit.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delbutton5">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Eliminar</string>
       </property>
       <property name="icon">
        <iconset resource="pandasoft.qrc">
         <normaloff>:/images/edit-delete.png</normaloff>:/images/edit-delete.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>350</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="volverboton">
      <property name="toolTip">
       <string>Volver a la Pantalla Principal</string>
      </property>
      <property name="statusTip">
       <string>Volver a la Pantalla Principal</string>
      </property>
      <property name="text">
       <string>Volver</string>
      </property>
      <property name="icon">
       <iconset resource="pandasoft.qrc">
        <normaloff>:/images/go-previous.png</normaloff>:/images/go-previous.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="pandasoft.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>volverboton</sender>
   <signal>clicked()</signal>
   <receiver>Gproductos</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
